src/database.cc

```diff
@@ -50,19 +50,6 @@ Database::Database(const std::string db_path)
 
     session_ = new Poco::Data::Session("SQLite", db_path);
 
-    {
-        int is_sqlite_threadsafe = sqlite3_threadsafe();
```

Comment:
IMO, I don't think that we should remove this assertion for couple reasons:
1. This logic will affect all other platforms: Although it's useless in Windows, but it might be not in linux or macOS.
2. This removal is out of scope of this ticket, which is 64-bit config (not refactor)

If we would like to remove it (especially in Library), I suggest that we could open ticket, then we will evaluate it carefully.

Reply:
Agree that it's out of scope, will revert these changes

Reply:
Done